pandas/core/internals/array_manager.py

```diff
@@ -754,10 +754,12 @@ def reindex_indexer(
         # ignored keywords
         consolidate: bool = True,
         only_slice: bool = False,
+        # ArrayManager specific keywords
+        do_integrity_check: bool = True,
```

Comment:
i think we call this integrity_check elsewhere

Reply:
The BlockManager/ArrayManager constructors call this `do_integrity_check`, so it's consistent with that. We don't have `integrity_check`, but we do have `verify_integrity` elsewhere (and this is actually used more, also in public APIs). 

I think it might be good to change `do_integrity_check` to `verify_integrity`, but given that currently `do_integrity_check` is consistently used *within* the (already existing) internals code, that's something for a different PR.

Reply:
right that's what i mean. can you normalize all to verify_integrity

Reply:
> can you normalize all to verify_integrity

Since that's not strictly related to concat or ArrayManager, will do that in a follow-up PR

Reply:
-> https://github.com/pandas-dev/pandas/pull/39989